### DISCUSSION TITLE
More than 50+ items in a single grid fetching data from backend

### DISCUSSION THREAD

**Discussion by jayarjo** [react-grid-layout/react-grid-layout#1696] [Q&A]
In our case every item in the grid is a widget that attempts to retrieve a data from the backend on mount. Obviously this is not a good idea when there are more than 50 of such widget on a single page. What is current approach to a problem like this with `react-grid-layout`?

**Comment by davidmoreno92** [comment]
I dont think that's a bad idea. Just persist the json with the data ready to print in database. 

You can just update the data in the JSON from your DB (with triggers) and let your UI just print data.
